pkg/encoding/bigint/bigint.go

```diff
@@ -15,6 +15,8 @@ const (
 	wordSizeBytes = bits.UintSize / 8
 )
 
+var bigOne = big.NewInt(1)
```

Comment:
We have the same (or `big.NewInt(0)`) var declarations in several places (i.g. for native contracts, stackitems and VM code), so maybe create a separate helper package for them?

Reply:
Not all of them can legitimately be reused, in some cases we do need a separate integer every time we use it. But if they can, then `bigint` probably is the right package for them. I'll check which ones can be used this way.

Reply:
After looking at it for while I'd say that private constants still make a bit more sense:
 * less chances things will go wild if something changes the variable
 * packages have only things they need (no useless constants or functionality from `bigint`)

Sure, `big.NewInt(1)` is a widely used thing, but 3 or 100 not so much. There will be some duplicates, but they don't cost a lot.